### ISSUE TITLE
CSpell Complains When No Spell-Checked Files Were Modified

### ISSUE THREAD

**Issue by Kurt-von-Laven** [oxsecurity/megalinter#2271]
**Describe the bug**
When MegaLinter is run via the `megalinter-incremental` pre-commit hook and CSpell has no work to do, CSpell errors.

**To Reproduce**

1. [Configure MegaLinter to run `megalinter-incremental` pre-commit in a test repository](https://megalinter.io/latest/mega-linter-runner/#pre-commit-hook)
2. [Install pre-commit](https://pre-commit.com/#install) via your preferred Python package manager.
3. Run `pre-commit install` in the test repository to install pre-commit hooks.
4. Commit your changes.
5. Modify a file that is either explicitly ignored in your CSpell configuration or that CSpell knows to ignore, such as `poetry.lock`.
6. Attempt to commit, yielding the following error.

```
Run MegaLinter (skipping linters that run in project mode).......................Failed
- hook id: megalinter-incremental
- exit code: 1

npm WARN exec The following package was not found and will be installed: mega-linter-runner@6.13.0
Pulling docker image oxsecurity/megalinter-documentation:v6.13.0 ... 
INFO: this operation can be long during the first use of mega-linter-runner
The next runs, it will be immediate (thanks to docker cache !)
v6.13.0: Pulling from oxsecurity/megalinter-documentation
Digest: sha256:6b28f2ad48125a65ce7233f55478e05db50b4f904b82957e4dcdadb50f339488
Status: Image is up to date for oxsecurity/megalinter-documentation:v6.13.0
docker.io/oxsecurity/megalinter-documentation:v6.13.0
Command: docker run --rm --name megalinter-incremental -v /var/run/docker.sock:/var/run/docker.sock:rw -v .../ScribeMD/template-repo:/tmp/lint:rw -e APPLY_FIXES=all -e LOG_LEVEL=warning -e SKIP_CLI_LINT_MODES=project -e MEGALINTER_FILES_TO_LINT=poetry.lock oxsecurity/megalinter-documentation:v6.13.0
Skipped setting git safe.directory DEFAULT_WORKSPACE:  ...
Setting git safe.directory default: /github/workspace ...
Setting git safe.directory to /tmp/lint ...
[MegaLinter init] ONE-SHOT RUN
[config] /tmp/lint/.mega-linter.yaml + Environment variables
MARKDOWN_REMARK_LINT has been temporary disabled in MegaLinter, please use a previous MegaLinter version or wait for the next one !
Unable to get number of errors with regex_number and Issues found: ([0-9]+) in .* files
❌ Linted [SPELL] files with [cspell]: Found 1 error(s) - (0.68s)
--Error detail:
CSpell: Files checked: 0, Issues found: 0 in 0 files

::set-output name=has_updated_sources::0
❌ Error(s) have been found during linting
To disable linters or customize their checks, you can use a .mega-linter.yml file at the root of your repository
More info at https://oxsecurity.github.io/megalinter/6.13.0/configuration/
```

**Expected behavior**
Finding 0 issues in 0 files should instead be treated as a success.

**Additional context**
It would be nice to add the following [arguments](https://cspell.org/docs/getting-started/#options) to CSpell by default: `--show-context --show-suggestions --no-must-find-files`, and the last of these would fix this issue. I am happy to make this change if others agree, but I am confused as to where to add these arguments.

**Comment by nvuillam** [comment]
I'm ok with everything but the --relative, because for now MegaLinter sends absolute paths with list-of-files cli_lint_mode, so it will break ci use :/

I really hope that someday i'll find the solution to the bug that make docker memory explode during test job in relative paths PR :(

**Comment by Kurt-von-Laven** [comment]
Thank you! I removed `--relative` from the issue to preclude the temptation to copy and paste it. Where would I add the other arguments? I found [`CSpellLinter.build_lint_command()`](https://github.com/oxsecurity/megalinter/blob/8b0074198cff990e42503700d79d7dfd78a11900/megalinter/linters/CSpellLinter.py#L21-L42), but that doesn't seem quite right.

Yeah, that is quite a tricky bug. Maybe even a sign of a bug in Docker itself?
